vespalib/src/vespa/vespalib/datastore/array_store.hpp

```diff
@@ -157,6 +157,20 @@ class CompactionContext : public ICompactionContext {
             }
         }
     }
+    void compact(vespalib::ArrayRef<AtomicEntryRef> refs) override {
```

Comment:
Consider if we should have an explicit test for this new function in vespalib/src/tests/datastore/array_store/array_store_test.cpp.

Reply:
Done.